.gitignore

```diff
@@ -100,3 +100,4 @@ postcss_error.log
 
 # ripgrep ignore file
 .rgignore
+config/definitions.rb
```

Comment:
This doesn't strictly speaking belong here but I use this to get better autocomplete with the language server and it's here already so maybe we can just leave it in.